### DISCUSSION TITLE
How to handle bounces with Mautic and Postal

### DISCUSSION THREAD

**Discussion by ashokvas** [postalserver/postal#2040] [Help with using Postal]
I have a Muatic + Postal SMTP server

I am not able to set up the bounce handling as I understand that Postal is sending the bounced emails to a different return path and Mautic is NOT able to read the inbox 

Has anyone  set up Mautic with Postal and been able to handle the bounces properly?

I have set up a webmail [bounce@mydomain.com](mailto:bounce@mydomain.com). and a separate segment in Muatic called "Bounced emails"

I am trying to monitor the inbox for bounces 

I need the following functionality

1) All bounced emails should come into my [bounce@mydomain.com](mailto:bounce@mydomain.com) mailbox
2) All bounced email addresses should be moved to the "Bounced email" segment
3) No further emails to be sent to the Bounced emails

I have a Postal SMTP connected to a Mautic system

Is this possible?

Can you help me with this?

**Comment by CarlosFongo** [comment]
> I have a Muatic + Postal SMTP server
> 
> I am not able to set up the bounce handling as I understand that Postal is sending the bounced emails to a different return path and Mautic is NOT able to read the inbox
> 
> Has anyone set up Mautic with Postal and been able to handle the bounces properly?
> 
> I have set up a webmail [bounce@mydomain.com](mailto:bounce@mydomain.com). and a separate segment in Muatic called "Bounced emails"
> 
> I am trying to monitor the inbox for bounces
> 
> I need the following functionality
> 
> 1. All bounced emails should come into my [bounce@mydomain.com](mailto:bounce@mydomain.com) mailbox
> 2. All bounced email addresses should be moved to the "Bounced email" segment
> 3. No further emails to be sent to the Bounced emails
> 
> I have a Postal SMTP connected to a Mautic system
> 
> Is this possible?
> 
> Can you help me with this?

Hi @ashokvas , did you end up finding a solution?
